src/components/numbered-text-section/numbered-text-section.tsx

```diff
@@ -21,7 +21,7 @@ export const NumberedSection: FC<Props> = ({ number, title, content }) => {
         <Typography variant="body1" sx={styles.title}>
           {title}
         </Typography>
-        {content}
+        <>{content}</>
```

Comment:
delete useless tags

Reply:
That tag is actually needed. In this case content is of type `ReactNode` which accepts an array of components (for example in `game-rules.tsx` content is an array of `<Bullet-point />`).

Since content is in a flex div with title, not grouping content into a component would cause alignment issues

Reply:
Makes sense, although now I wonder why it was working before lol

Reply:
to be clear `ReactNode` can also be a single component, in which case that tag is indeed useless 😅